contrib/spec/podman.spec.in

```diff
@@ -502,6 +474,7 @@ export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
 %license LICENSE
 %doc README.md CONTRIBUTING.md install.md code-of-conduct.md transfer.md
 %{_bindir}/%{name}
+%{_bindir}/py%{name}
```

Comment:
sorry about suggesting this earlier. This line should actually be in the python-podman or python-pypodman (if we're doing separate rpms for these)

Reply:
Yes lets keep pypodman in the python3-podman package.